hbase_sql/src/main/java/com/splicemachine/stream/StreamableRDD.java

```diff
@@ -153,6 +156,8 @@ private void submitBatch(int batch, int batchSize, int numPartitions, final Java
             public Object call() throws Exception {
                 SparkContext sc = SpliceSpark.getContextUnsafe().sc();
                 sc.setLocalProperties(properties);
+                if (shufflePartitions != null)
+                    sc.setLocalProperty(SQLConf.SHUFFLE_PARTITIONS().key(), shufflePartitions.toString());
```

Comment:
Why do we need to set shuffle partition property both here and in QueryJob?

Reply:
This wasn't needed, it was an initial attempt at fixing the problem that didn't work out, nice catch!